docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,186 @@
+`my-elser-model`.
+<2> The `elser` service is used in this example.
+
+
+[discrete]
+[[semantic-text-index-mapping]]
+==== Create the index mapping
+
+The mapping of the destination index - the index that contains the embeddings
+that the model will create based on your input text - must be created. The
+destination index must have a field with the <<semantic-text,`semantic_text`>>
+field type to index the output of the used model.
+
+[source,console]
+------------------------------------------------------------
+PUT semantic-embeddings
+{
+  "mappings": {
+    "properties": {
+      "semantic_text": { <1>
+        "type": "semantic_text", <2>
+        "inference_id": "my-elser-model" <3>
+      },
+      "content": { <4>
+        "type": "text",
+        "copy_to": "semantic_text" <5>
+      }
+    }
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The name of the field to contain the generated tokens.
+<2> The field to contain the tokens is a `semantic_text` field.
+<3> The `inference_id` is the inference endpoint you created in the previous step.
+It will be used to generate the tokens based on the input text.
+Every time you ingest data into the related `semantic_text` field, this endpoint will be used for creating the vector representation of the text.
+<4> The field to store the text reindexed from a source index in the  <<semantic-text-reindex-data,Reindex the data>> step.
+<5> The textual data stored in the `content` field will be copied to `semantic_text` and processed by the {infer} task.
+The `semantic_text` field will store the tokens generated based on the input data.
+
+
+[discrete]
+[[semantic-text-load-data]]
+==== Load data
+
+In this step, you load the data that you later use to create embeddings from it.
+
+Use the `msmarco-passagetest2019-top1000` data set, which is a subset of the MS
+MARCO Passage Ranking data set. It consists of 200 queries, each accompanied by
+a list of relevant text passages. All unique passages, along with their IDs,
+have been extracted from that data set and compiled into a
+https://github.com/elastic/stack-docs/blob/main/docs/en/stack/ml/nlp/data/msmarco-passagetest2019-unique.tsv[tsv file].
+
+Download the file and upload it to your cluster using the
+{kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[Data Visualizer]
+in the {ml-app} UI. Assign the name `id` to the first column and `content` to
+the second column. The index name is `test-data`. Once the upload is complete,
+you can see an index named `test-data` with 182469 documents.
+
+
+[discrete]
+[[semantic-text-reindex-data]]
+==== Reindex the data
+
+Create the embeddings from the text by reindexing the data from the `test-data`
+index to the `semantic-embeddings` index. The data in the `content` field will
+be reindexed into the `content` field of the destination index. Then the same
+data will be copied to the `inference_content` field as a result of the
```

Comment:
This is inconsistent, the field name is `semantic_text` earlier in this doc. I like `inference_content` as a field name better (because it's different than the name of the field type), can we change to use `inference_content` throughout the doc?

Reply:
It was `inference_content` throughout the docs, then it was [requested](https://github.com/elastic/elasticsearch/pull/108311#discussion_r1610191161) to change it to `semantic_text` to simplify it. I just missed this occurrence.